Makefile

```diff
@@ -80,7 +80,7 @@ generate: generate-go generate-deploy generate-test
 goveralls: $(GOVERALLS)
 	GOVERALLS=$(GOVERALLS) ./hack/goveralls.sh
 
-manager: $(GO)
+manager: generate-go
```

Comment:
I think this should be `generate` since `generate-go` does not create the manifests..

Reply:
It generates go code

Reply:
but don't we want to also generate the manifests files?

Reply:
this just depends on minimal stuff needed to compile manager and that is the generated golang code the manifests are needed for deploy

Reply:
ack